beets/autotag/mb.py

```diff
@@ -447,6 +454,12 @@ def album_info(release):
     if config['musicbrainz']['genres'] and genres:
         info.genre = ';'.join(g['name'] for g in genres)
 
+    # supplementary tags provided by plugins
+    extra_albumdatas = plugins.send('extracting_albumdata', info=release)
```

Comment:
For the names of the events, I think it would be good to clarify that this is for MusicBrainz specifically. So perhaps `mb_album_extract` or similar?